torchdata/stateful_dataloader/stateful_dataloader.py

```diff
@@ -479,7 +484,11 @@ def __init__(self, loader, next_iter_state=None):
             self.load_state_dict(next_iter_state)
         else:
             self._dataset_fetcher = _DatasetKind.create_fetcher(
-                self._dataset_kind, self._dataset, self._auto_collation, self._collate_fn, self._drop_last
+                self._dataset_kind,
+                self._dataset,
+                self._auto_collation,
+                self._collate_fn,
+                self._drop_last,
```

Comment:
nit: can we remove the whitespace changes (ie run pre-commit to format all these files) so it's easier to review?

Reply:
I have run precommit and these whitespaces are actually added by `precommit`. If I revert back everything I get lint errors.